boards/arm/mps3_an547/mps3_an547.dts

```diff
@@ -132,6 +132,7 @@
 	/* DDR4 - 2G, alternates non-secure/secure every 256M */
 	ddr4: memory@60000000 {
 		device_type = "memory";
```

Comment:
First commit seems to be doing more than adding a binding, please, split commits properly.

Reply:
@kristofer-jonsson-arm if you need help splitting commits let us know, we can help.

Reply:
Fixed